skimage/exposure/_adapthist.py

```diff
@@ -70,7 +70,7 @@ def equalize_adapthist(image, ntiles_x=None, ntiles_y=None, clip_limit=0.01,
 
     References
     ----------
-    .. [1] http://tog.acm.org/resources/GraphicsGems/gems.html#gemsvi
+    .. [1] http://tog.acm.org/resources/GraphicsGems/
```

Comment:
Why not `http://www.realtimerendering.com/resources/GraphicsGems/gems.html#gemsiv`?